app/views/layouts/application.html.erb

```diff
@@ -73,9 +73,12 @@
         <div id="audiocontent" data-podcast="">
           <%= yield(:audio) %>
         </div>
-        <%= render "layouts/top_bar" %>
-        <div id="active-broadcast" class="broadcast-wrapper"></div>
-        <div id="message-notice"></div>
+        <% if !creator_onboarding_mode %>
```

Comment:
Nitpick: `unless` is preferred over `if !` in Ruby as long as there's no `else` branch.

```suggestion
        <% unless creator_onboarding_mode %>
```